src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { Post } from '../../types/Post';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
user component has to be rendered, only when user is present

Reply:
Done ;)